tests/rados/scheduled_scrub_scenarios.py

```diff
@@ -52,11 +27,18 @@ def run(ceph_cluster, **kw):
     cephadm = CephAdmin(cluster=ceph_cluster, **config)
     rados_obj = RadosScrubber(node=cephadm)
     mon_obj = MonConfigMethods(rados_obj=rados_obj)
-    # Storing the pg dump log before setting the scrub parameters
-    before_scrub_log = rados_obj.get_pg_dump("pgid", "last_scrub_stamp")
-    before_deep_scrub_log = rados_obj.get_pg_dump("pgid", "last_deep_scrub_stamp")
-    # Preparation of cofiguration parameter values from the current
-    # cluster time
+    # Creating ec pool
+    ec_config = config.get("replicated_pool")
```

Comment:
Looks like we are passing replicated pool configs here. Can we rename the param?

Or are we intending to pass EC config itself?